electrum/gui/qt/qrtextedit.py

```diff
@@ -14,10 +14,10 @@ def __init__(self, text=None, *, config: SimpleConfig):
         self.config = config
         self.setReadOnly(True)
 #todo uncomment when QR Read gonna be fixed
-#        icon = "qrcode_white.png" if ColorScheme.dark_scheme else "qrcode.png"
-#        self.addButton(icon, self.qr_show, _("Show as QR code"))
-#
-#        run_hook('show_text_edit', self)
+        icon = "qrcode_white.png" if ColorScheme.dark_scheme else "qrcode.png"
+        self.addButton(icon, self.qr_show, _("Show as QR code"))
+
+        run_hook('show_text_edit', self)
```

Comment:
if its working, remove the comment please

Reply:
done